crates/edr_provider/src/data.rs

```diff
@@ -2774,6 +2774,16 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn fork_metadata_fork_mode() -> anyhow::Result<()> {
+        let fixture = ProviderTestFixture::new_forked(None)?;
+
+        let fork_metadata = fixture.provider_data.fork_metadata().unwrap();
+        assert_eq!(fork_metadata.chain_id, 1);
```

Comment:
Is it ok to assume that we are forking mainnet here? I think it is, but would like to get confirmation.

If it's not ok, is there an example of a test where we are doing this "correctly"?

Reply:
We assume this everywhere else in the code, so I think it's fine.